samples/cisd-h2o-clpse/test.in

```diff
@@ -15,14 +15,14 @@ molecule h2o {
 
 set {
   basis 6-31G**
+  qc_module detci
 }
 
 set detci {
   guess_vector = UNIT
   r_convergence = 5
   max_num_vecs = 4
   collapse_size = 2
-  qc_module detci
```

Comment:
indeed. i was scared that the whole thing was broken. but the option just looks globally, since which local options set is unclear.